src/envo/envo-edit.owl

```diff
@@ -39978,6 +39980,32 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/ENVO_00000015>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_01001497>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105004> (thermosetting disposition)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Thermosetting_polymer>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105004> "A disposition inhering in a bearer by virtue of the bearer's ability to become permanently rigid when heated or cured."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105004> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105004> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105004> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105004> "2021-04-27T13:46:52Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105004> "熱固性"@zh)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105004> "envoPlastics"@zh)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105004> "thermosetting disposition"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105004> <http://purl.obolibrary.org/obo/BFO_0000016>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105005> (thermosetting polymer)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Thermosetting_polymer>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105005> "A polymer which is irreversibly hardened by curing from a soft solid or viscous liquid prepolymer or resin."@en)
```

Comment:
>A polymer which is irreversibly hardened by curing from a soft solid or viscous liquid prepolymer or resin.

Careful with the tenses:

>A polymer which may be irreversibly hardened by heat-induced curing. 


Add as a comment: 

> Prior to curing, thermosetting polymers often occur as a soft solid or viscous liquid prepolymer or resin.

Reply:
Hi @pbuttigieg, I don't quite see the reason of changing `is` to `may be`. Could you explain a bit more?

Reply:
@pbuttigieg 

Do you propose changing the tenses to also include the not yet hardened thermosetting material? When consulting the literature, I noticed that thermosetting polymer is usually used to refer to the end product (which has been hardened by curing) while thermosetting resin is referring to the material that has not yet undergone the hardening process.

Reply:
@pbuttigieg 

We've now updated the definition and comment:

Definition 

> A polymer which is irreversibly hardened by heat-induced curing.

Comment

> Prior to curing, thermosetting materials often occur as a soft solid or viscous liquid prepolymer or resin. In the literature, those are often referred to thermosetting resins.
During curing process, well-defined and irreversible chemical networks form in three dimensions thus giving the thermosets a strong and rigid structure that can be added to other materials to increase strength.

We also add `thermosetting material` as broad synonym as it can refer to both thermosetting polymer or thermosetting resin.

Please let us know if this is ok.